index.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/extensions */
+
+import Storage, {
+  Card,
+  arrBooks,
+  bookTitle,
+  bookAuthor,
+} from './modules/storage.js';
+import Books from './modules/books.js';
+import { DateTime } from './modules/luxon.js';
+/* eslint-disable no-unused-vars */
+const bodyCont = document.body;
+const form = document.querySelector('#form');
+const tableContainer = document.querySelector('.cards');
+const warningLabel = document.querySelector('.warningLabel');
+const navbar = document.querySelector('.navbar-nav');
+const booksList = document.querySelector('.books-list');
+const addNew = document.querySelector('.add-new');
+const contact = document.querySelector('.contact');
+const navlist = document.querySelector('.navlist');
+const navadd = document.querySelector('.navadd');
+const navcontact = document.querySelector('.navcontact');
+
+const d = DateTime.local().toLocaleString(DateTime.DATETIME_FULL);
+
+document.querySelector('.date-display').innerHTML = d;
+
+let id = arrBooks.length;
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] Kindly remove eslint disabler. You can solve this issue by changing i++ to i + 1.